backend/fms_core/template_importer/row_handlers/sample_submission/sample.py

```diff
@@ -129,6 +129,11 @@ def process_row_inner(self, sample, library, container, project, parent_containe
                                                                                            index=index_obj,
                                                                                            platform=platform_obj,
                                                                                            strandedness=library['strandedness'])
+
+        # Check if there's a sample with the same name
+        if Sample.objects.filter(name=sample['name']).exists():
+            self.warnings['name'] = f'Sample with the same name [{sample["name"]}] already exists.'
```

Comment:
I would explain what will be the resulting action taken if they keep going ("A new sample with the same name will be created.)